src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// Write code here
+import React from 'react';
+import './Person.scss';
+
+export function Person({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) {
+  let partnerSex = 'My husband`s name is ';
+
+  if (sex === 'm') {
+    partnerSex = 'My wife`s name is ';
+  }
```

Comment:
Don't repeat yourself, all you need is to change one word in a string, so create a variable for this word (you can use ternary) and then use it in your string